spec/web/controllers/auth_controller_spec.exs

```diff
@@ -42,6 +42,17 @@ defmodule Squadster.Web.AuthControllerSpec do
               expect(redirect_url =~ "auth_token") |> to(be true)
               expect(redirect_url =~ "image_url") |> to(be true)
             end
+
+            it "should not create user" do
+              initial_count = entities_count(User)
+              success_conn() |> AuthController.callback(%{}) |> redirected_to(302)
+              expect(entities_count(User)) |> to(eq(initial_count))
+            end
+
+            it "includes show_info: false" do
+              redirect_url = success_conn() |> AuthController.callback(%{}) |> redirected_to(302)
+              expect(redirect_url =~ "show_info=false") |> to(be true)
```

Comment:
`expect(redirect_url) |> to(match "show_info=false")`

Reply:
And refactor remaining expectations with that matcher

Reply:
Nice, will fix